### PR TITLE
Automatically build when CSS files change

### PR DESCRIPTION
**Pull request by stevage**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -69,6 +69,7 @@ gulp.task('release', ['build-css', 'merge-datasources', 'release-app', 'release-
 
 gulp.task('watch-app', ['prepare'], function() {
     return watch(appJSName, appEntryJSName, false);
+    // TODO: make this automatically trigger when ./lib/Views/*.html get updated
```
Comment (kring) [inline gulpfile.js]: I think watchify does this automatically.

Comment (stevage) [inline gulpfile.js]: No, it doesn't.

Comment (kring) [inline gulpfile.js]: Hmm I just tried it.  I changed RelatedMaps.html, saved it, and my `gulp watch-app` noticed that change and rebuilt nationalmap.js.  Maybe I don't understand what you're referring to?

Comment (stevage) [inline gulpfile.js]: Heh. It didn't when I tried it, which was why I left the comment. Anyway...

**Pull request merged**